services.yaml

```diff
@@ -909,49 +926,197 @@ paths:
           description: NOT FOUND
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+    get:
+      tags:
+        - View Folders
+        - VFs
+      summary: "List View Folders"
+      description: "List View Folders"
+      responses:
+        200:
+          description: List of VF names
+          schema:
+            properties:
+              vf_names:
```

Comment:
this sounds like something engineering are using, but it's not a standard as we shouldn't be putting abbrevations in the keys. Also this can't be "names" as we're discussing a full Folder object, so it should be an array of Folders, and that's it.

Reply:
renamed to 'folders' to be consistent with current content. If we decide on a different user facing named, i will rename all.